cpp/daal/src/threading/threading.h

```diff
@@ -337,14 +352,23 @@ class static_tls
             _storage[i] = nullptr;
         }
 
-        const void * ac = static_cast<const void *>(&lambda);
+        lambdaType * locall = new lambdaType(lambda);
+        _deleter            = new static_tls_deleter_<lambdaType>();
```

Comment:
Now, ```locall``` and ```_deleter``` can be ```nullptr``` due to potential memory allocation fail.
Need to handle this somehow.

Reply:
it was fixed